src/main/protobuf/command_server.proto

```diff
@@ -58,6 +58,10 @@ message RunRequest {
   // came from. These options have already been parsed and already have had
   // their effect. This information should only be used for logging.
   repeated StartupOption startup_options = 6;
+
+  // Whether the resulting command should be preempted if additional commands
```

Comment:
s/should/can/ - calling this one out explicitly since it's in the grpc interface. Also, don't forget '.' at the end of the sentence.